docker/src/config.ts

```diff
@@ -87,8 +87,10 @@ const getApiConfig = (apiUrl: string): ApiClientConfig => {
     "GATLING_ENTERPRISE_API_TOKEN is required"
   );
   return {
-    baseUrl: `${apiUrl}/api/public`,
-    apiToken
+    baseUrl: apiUrl,
+    apiToken: apiToken,
+    pluginFlavor: PluginFlavor.ENTERPRISE_RUNNER,
+    pluginVersion: require('package.json').version
```

Comment:
I made this, but I wonder if this is the right way to do it. Does someone see something better or do we keep this ?